.github/workflows/build_test.yml

```diff
@@ -0,0 +1,44 @@
+
+name: Build Test
+on: [push]
+
+jobs:
+  check_format:
+    runs-on: ubuntu-latest
+    container: px4io/px4-dev-ros-kinetic:2019-10-24
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        token: ${{ secrets.ACCESS_TOKEN }}
+    - name: check_format
+      working-directory: tools
+      run: |
+        apt update
+        apt install -y clang-format-3.8
+        ./check_code_format.sh
+  build:
+    needs: check_format
+    runs-on: ubuntu-latest
+    container: px4io/px4-dev-ros-melodic:2019-10-24
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        token: ${{ secrets.ACCESS_TOKEN }}
+    - name: release_build_test
+      working-directory: 
+      run: |
+        mkdir -p $HOME/catkin_ws/src;
+        cd $HOME/catkin_ws
+        catkin init
+        catkin config --extend "/opt/ros/melodic"
+        catkin config --merge-devel
+        cd $HOME/catkin_ws/src
+        ln -s $GITHUB_WORKSPACE
+        catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release -DCATKIN_ENABLE_TESTING=False -DDISABLE_SIMULATION=ON
+        catkin build -j$(nproc) -l$(nproc) local_planner
+    - name:
+      working-directory:
+      run: |
+        cd $HOME/catkin_ws/src
+        catkin config --cmake-args -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCATKIN_ENABLE_TESTING=True
+        catkin build avoidance local_planner --no-deps -v -i --catkin-make-args tests
```

Comment:
This is building the  tests, but doesn't run them

Reply:
@jkflying I am waiting for valgrind to be included in the container

Reply:
Ok, we can still run the tests, just not in valgrind though.

Reply:
Tests now running :)